src/ebook-speaker.c

```diff
@@ -711,7 +711,7 @@ void view_screen (misc_t *misc, daisy_t *daisy)
       if (l / 2 * 2 == l)
          waddstr (misc->screenwin, " ");
       for (x = l; x < 56; x += 2)
-         waddstr (misc->screenwin, " .");
+         waddstr (misc->screenwin, "  ");
```

Comment:
if we are going to display only spaces, you can replace this 2-by-2 loop and the previous if with a single 1-by-1 loop.

Reply:
Yes we can do that.